mempool/metrics.go

```diff
@@ -36,9 +36,9 @@ func PrometheusMetrics(namespace string) *Metrics {
 			Subsystem: MetricsSubsytem,
 			Name:      "tx_size_bytes",
 			Help:      "Transaction sizes in bytes.",
-			Buckets:   stdprometheus.ExponentialBuckets(1, 2, 10),
+			Buckets:   stdprometheus.ExponentialBuckets(1, 2, 24),
```

Comment:
Do you think the curve should be more steep with less number of buckets? (1, 3, 17) => 9. 27, 81, 243, 729, 2KB, 6KB, 19KB, 59KB, 177KB, 531KB, 1.5MB, 4,7MB

Reply:
Sure - I'm unfamiliar with the best bucketing strategy, so will happily defer to you.